### PR TITLE
Prepare release 34

### PR DESCRIPTION
**Pull request by diaasami**
Tasks:
- [x] Bump version
- [x] Update news

### CODE REVIEW

**Review by Gundersanne — APPROVED** [review]
:exploding_head:

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## CHANGES WITH 34:
+
+  * Support temporary AWS credentials
+  * Composer-api and worker-api: OAuth2 support
+  * Install docs in RHEL 8.5 and 9.0 images
+  * RHEL-8.5 / RHEL-9.0: RHSM DNF plugins are now enabled by default on `ec2` and `ami` images
+  * Add support for new osbuild stages required for RHEL EC2 SAP images
+
+Contributions from: Alexander Todorov, Antonio Murdaca, Brian C. Lane, Chloe Kaubisch, Diaa Sami, Gianluca Zuccarelli, Jakub Rusz, Juan Abia, Martin Sehnoutka, Ondřej Budai, Simon Steinbeiss, Thomas Lavocat, Tom Gundersen, Tomas Hozza, Sanne Raymaekers, Yi He
```
Comment (ondrejbudai) [inline NEWS.md:9]: Can you format this so the line length is <= 80 characters, please? Just so we are consistent. Also line 6 might be an issue.

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
+## CHANGES WITH 34:
```
Comment (ondrejbudai) [inline NEWS.md:1]: Can you bring line 22 on top of this file? This was my fault when I did the last release and I forgot to fix it, sorry. :(

**Review by ondrejbudai — COMMENTED** [review]
An awesome release, two minor things inline.

**Review by ondrejbudai — DISMISSED** [review]
Thanks! :)

**Review by ondrejbudai — APPROVED** [review]

**Pull request merged**